.github/workflows/update-integration-tests.yml

```diff
@@ -96,3 +96,4 @@ jobs:
           test_folder: ui-tests
           npm_client: jlpm
           server_url: http-get://localhost:3000
+          browser: chromium
```

Comment:
This is required due to https://github.com/jupyterlab/maintainer-tools/pull/256 which was released last week in https://github.com/jupyterlab/maintainer-tools/releases/tag/v0.28.1